<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librepcb::editor::HolePropertiesDialog</class>
 <widget class="QDialog" name="librepcb::editor::HolePropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hole Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="librepcb::editor::HoleEditorWidget" name="holeEditorWidget" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>librepcb::editor::HoleEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">librepcb/editor/widgets/holeeditorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
